.circleci/config.yml

```diff
@@ -180,6 +180,33 @@ jobs:
       - *store_test_package_artifacts
       - *store_package_artifacts
 
+  uat_tests:
+    <<: *container_config
+    steps:
+      - checkout
+      - *restore_rebar_cache
+      - *restore_build_cache
+      - run:
+          name: UAT Tests
+          command: |
+            epmd -daemon
+            make python-env && make swagger && make multi-build
```

Comment:
Why `make swagger`?

Reply:
@lucafavatella I followed the docs here: https://github.com/aeternity/epoch/tree/master/py

If the docs are wrong, let's update them in this PR too.